server/db/database.js

```diff
@@ -0,0 +1,8 @@
+// This module is to establish a connection to the Postgres database
+// by creating a Sequelize instance (called "db").
+const Sequelize = require('sequelize');
+const db = new Sequelize('postgres://localhost:5432/grace-shopper', {
+    logging: false,
+});
+
+module.exports = { db };
```

Comment:
we don't need to use the " { }"  because we're bringing sequelize as library to define the variable db, so we just only need to export db; 

module.exports = db;